org.opent2t.sample.lamp.superpopular/com.wink.lightbulb/js/thingTranslator.js

```diff
@@ -479,7 +548,7 @@ function providerSchemaToPlatformSchema(providerSchema, expand) {
         dimmingRange: [0,100],
```

Comment:
Good point - I'd like the range to be the real range but we'll need to keep this one as upscaled since the app uses integers to set these values.